jest-playwright.config.js

```diff
@@ -2,14 +2,20 @@ module.exports = {
 
     // Describe which browsers we want to run
 
-    browsers: ["chromium", "firefox", "webkit"],
+    browsers: ["chromium", "webkit", "firefox"],
     exitOnPageError: false,
     launchOptions: {
+
```

Comment:
Was these white space changes done by linter? If not, I suggest to not add them.

Reply:
Removed